api/src/main/webapp/static/activity-checker-extension.js

```diff
@@ -0,0 +1,45 @@
+// Sends a message to the parent frame every second that user activity is detected
```

Comment:
We should either remove the files from /cluster-resources, or copy the files here as part of the build step and .gitignore these. Currently the file is duplicated in source.

Reply:
I can do the copy. There was a requirement to keep them served to GCS, so I didn't want to mess with that code.